inc/version_decl

```diff
@@ -1 +1 @@
-   CHARACTER (LEN=10) :: release_version = 'V4.1      '
+   CHARACTER (LEN=10) :: release_version = 'V4.1.1      '
```

Comment:
Here we're assigning a string of length 12 to a variable of declared length 10.

Reply:
I think it's valid Fortran to write something like this:
```fortran
CHARACTER (LEN=*), PARAMETER :: release_version = 'V4.1.1' 
```
This way, we don't need to worry about putting in the right number of spaces. However, I don't know whether there is downstream code that depends on `release_version` being exactly 10 characters or that needs to change `release_version`.

Reply:
@mgduda 
Michael,
Thanks a lot. i have changed the code as suggested.